### PR TITLE
HERE.OLPCLI version 10.5.106

### PR DESCRIPTION
**Pull request by ksen0byte**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56376)

### CODE REVIEW

File: manifests/h/HERE/OLPCLI/0.0.0/HERE.OLPCLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: HERE.OLPCLI
+PackageVersion: 0.0.0
```
Comment (ItzLevvie) [inline manifests/h/HERE/OLPCLI/0.0.0/HERE.OLPCLI.installer.yaml:5]:
```suggestion
PackageVersion: 10.5.106
```
This must match Control Panel.

File: manifests/h/HERE/OLPCLI/0.0.0/HERE.OLPCLI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HERE.OLPCLI
+PackageVersion: 0.0.0
```
Comment (ItzLevvie) [inline manifests/h/HERE/OLPCLI/0.0.0/HERE.OLPCLI.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 10.5.106
```
This must match Control Panel.

File: manifests/h/HERE/OLPCLI/0.0.0/HERE.OLPCLI.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: HERE.OLPCLI
+PackageVersion: 0.0.0
```
Comment (ItzLevvie) [inline manifests/h/HERE/OLPCLI/0.0.0/HERE.OLPCLI.yaml:5]:
```suggestion
PackageVersion: 10.5.106
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will also need to update the version folder from `0.0.0` to `10.5.106` using Visual Studio Online by pressing the `.`  (dot) on your keyboard on this PR.

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Wait for an engineer to take a look.

File: manifests/h/HERE/OLPCLI/10.5.106/HERE.OLPCLI.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HERE.OLPCLI
+PackageVersion: 10.5.106
+PackageLocale: en-US
+Publisher: HERE Technologies
+PackageName: OLP CLI
+License: Copyright (C) 2017-2022 HERE Global B.V. and its affiliate(s).
```
Comment (Trenly) [inline manifests/h/HERE/OLPCLI/10.5.106/HERE.OLPCLI.locale.en-US.yaml:9]: A copyright is not a license. Please move this to the `Copyright` field and provide a valid license

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request merged**